Userland/Libraries/LibWeb/Page/EditEventHandler.cpp

```diff
@@ -44,8 +44,8 @@ void EditEventHandler::handle_delete(JS::NonnullGCPtr<DOM::Document> document, D
 
     if (start == end) {
         StringBuilder builder;
-        builder.append(start->data().bytes_as_string_view().substring_view(0, range.start_offset()));
-        builder.append(end->data().bytes_as_string_view().substring_view(range.end_offset()));
+        builder.append(MUST(start->substring_data(0, range.start_offset())));
+        builder.append(MUST(end->substring_data(end->length(), range.end_offset())));
```

Comment:
Hmm, maybe I am mistaken, but should it technically be utf16 code units instead of by code points?

Reply:
We are not very consistent with this around the place, but I believe the spec says we should be using them as utf16 code unit offsets since lengths are also specified as such.

Reply:
I thought code points / units were the same thing. I'll look into it.